src/commands/app/shell.ts

```diff
@@ -69,7 +69,7 @@ export default class AppShell extends Command {
       duplex.pipe(process.stdout);
 
       process.stdin.on('data', function (key) {
-        if (key.toString() == CTRL_Q) {
+        if (key.toString() === CTRL_Q) {
           clearStdinEffects();
```

Comment:
I'm not sure why you used two equal signs here.

Reply:
I haven't. I guess Mohammad Amin added these lines.